apps/reader/views.py

```diff
@@ -105,9 +107,7 @@ def dashboard(request, **kwargs):
         del request.session['import_from_google_reader']
     
     if not user.is_active:
-        url = "https://%s%s" % (Site.objects.get_current().domain,
-                                 reverse('stripe-form'))
-        return HttpResponseRedirect(url)
+        return HttpResponseRedirect(reverse('stripe-form'))
```

Comment:
You'll want to revert this part. This is explicitly redirecting the user to the SSL version, which they may not be on. And we need to not only support http but encourage it as many other websites are also http-only and can't be loaded on an https-only NewsBlur.